src/traits.rs

```diff
@@ -6,7 +6,10 @@ pub trait TextSized: Copy {
     fn text_size(self) -> TextSize;
 }
 
+/// This will panic for strings larger than `TextSize::MAX` when
+/// debug assertions are enabled, and wrap when they are disabled.
```

Comment:
Hm, this is wrong? `try_into` will return `Err` even in release

Reply:
Yeah, that's a conflict from #20.